@here/harp-geojson-datasource/lib/GeoJsonDataProvider.ts

```diff
@@ -10,6 +10,19 @@ import { TileKey } from "@here/harp-geoutils";
 import { ConcurrentTilerFacade } from "@here/harp-mapview";
 import { DataProvider } from "@here/harp-mapview-decoder";
 
+export interface GeoJsonDataProviderOptions {
+    /**
+     * WorkerTilerUrl Worker tiler URL. Defaults to `./decoder.bundle.ts` in the
```

Comment:
> Worker script hosting [[TilerService]. 
> @default ...

1. Use @default and other tags
2. Don't repeat: like this: "WorkerTilerUrl Worker tiler URL"

Reply:
is that what you meant?
`    /**
     * Worker script hosting [[Tiler Service]]
     * @default `./decoder.bundle.ts`
     */
    workerTilerUrl?: string;`